Channels/Production/release_notes.md

```diff
@@ -1,28 +1,30 @@
-## June 18th, 2019 Production Release ([v1.1.0883.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0883.01))
+## August 5, 2019 Insider Release ([v1.1.0933.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0933.01))
```

Comment:
@Sirisha-Kommushetty, please double-check the release notes. They should match (except for the date and the channel) what's currently at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.0933.01